### ISSUE TITLE
[B] Jovo build --reverse give error

### ISSUE THREAD

**Issue by Mistic92** [jovotech/jovo-framework#724]
## I'm submitting a...

- [X] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior
Revert works

## Current Behavior
When using command ```jovo build --reverse``` on Dialogflow project I'm getting error. I was trying on project with two languages (pl+en) and with english only. 

### Error log
```
╰─ jovo build --reverse

 jovo build:  Create and update platform specific files in /platforms folder
   >> Learn more: https://jovo.tech/docs/cli/build

? Please select the platform you want to reverse build from: GoogleAction with DialogFlow (googleAction)
  ❯ Building language model platform model
    ❯ Reversing model files
      ✖ en
        → Cannot read property 'length' of undefined
 ›   Error: There was a problem:
 ›   TypeError: Cannot read property 'length' of undefined
```

## Your Environment

* Jovo Framework version used: 3.0.13
* Operating System: Windows 10 WSL2

**Comment by Mistic92** [comment]
I found an issue in your cli tool. It is expecting that language in Dialogflow agent package will use full locale like ```en-us``` while it's using ```en```. 

When I changed this to   "language": "en-us", it worked. 

I'm not sure where to look for this part of code to submit PR with fix.

**Comment by Mistic92** [comment]
I found that it's not importing training phrases as it's using ```en``` instead of ```en-us``` so it's incompatible too.

**Comment by rubenaeg** [comment]
Hey @Mistic92, what version of the Jovo CLI are you using?

**Comment by Mistic92** [comment]
3.0.13

**Comment by rubenaeg** [comment]
I mean the Jovo CLI. Try `jovo -v` outside of your project folder. It should print a message similar to this one: `Jovo CLI Version: 3.0.11`

**Comment by Mistic92** [comment]
Jovo CLI Version: 3.0.7

**Comment by rubenaeg** [comment]
Could you please update to the latest version using `npm install -g jovo-cli` and try again? We fixed a few problems since `3.0.7` and perhaps your issue was already resolved.

**Comment by aswetlow** [comment]
Closing due to inactivity. Please re-open if it still persists.
